app/controllers/settings/api_controller.rb

```diff
@@ -0,0 +1,5 @@
+module Settings
+  class APIController < ApplicationController
+    def show; end
+  end
+end
```

Comment:
Can you move this back into the `def api` on settings pls. I don't want a whole extra controller for it. You can rename it to `api_token` if there's a naming conflict.

Add also add `communication_preferences` as an action on `settings`, rather than having a separate controller pls.

Reply:
I've done this.